### PR TITLE
[X86] Recognize VPXORDZrr as a zero-idiom on Znver4

### PR DESCRIPTION
**Pull request by boomanaiden154**
This patch adds information about VPXORDZrr to the znver4 scheduling model, particularly that it is a zero-idiom.

This fixes a proximal cause of #108157.

### CODE REVIEW

**Comment by boomanaiden154** [comment]
I don't have hardware to test that this is actually a zero-idiom, but I would definitely expect it to be.

This is just a stop-gap for the linked issue. Once we have confirmation on the other AVX512 zero-idioms for `znver4`, we should add those in too as a follow-up step.

**Review by RKSimon — APPROVED** [review]
LGTM as a stopgap - after this has landed, please can you add a test file for #108157 as well? We can keep #108157 open until we have the real fix.

**Comment by adibiagio** [comment]
I plan to send a patch very soon. So, don't worry about adding a test file for it.
Thanks!

**Pull request merged**
